docs/design/10-management-networks.md

```diff
@@ -1,12 +1,12 @@
 # Management Networks
-Most leafpairs will have management networks living on them, which shall be used by the hypervisor solution in a
+Most leaf pairs will have management networks living on them, which shall be used by the hypervisor solution in a
 building block for whatever they are needed (VM migration, management communication, storage, console, ...).
-Originally these networks were seen as leafpair-local, but as we also want to be able to put servers from another
-leafpair into these networks (e.g. in case of maintenance or hardware failure), we cannot treat these networks as
-leafpair-local.
+Originally these networks were seen as local to a leaf pair, but as we also want to be able to put servers from another
+leaf pair into these networks (e.g. in case of maintenance or hardware failure), we cannot treat these networks as
+local to a leaf pair.
 
 Between members of this network we need to have VLAN consistency (as we do when "lending a host" to another bb).
 This means each port in a management network will get it as the same VLAN (i.e. mgmt --> 100). It will still need
 to be represented with its own VNI inside the fabric.
 
-Management networks are l3, so they also need to have a gateway.
+Management networks are L3, so they also need to have a gateway.
```

Comment:
Why uppercase `L` here? would we capitalize Layer 3?

Reply:
Because it's an acronym which we wrote in upper case in most places and to make it more readable and upper case L is easier distinguished from 1 and i as lower case :-)